### DISCUSSION TITLE
M1 Pro / Max

### DISCUSSION THREAD

**Discussion by DylanSpeiser** [nerdyrodent/VQGAN-CLIP#79] [Q&A]
Newbie here: I’m looking to run VQGAN-CLIP on my new M1 Max laptop. Does this repo work?

**Comment by antiero** [comment]
I don't think it works out of the box, but some exciting news on the PyTorch front announced yesterday which should in theory enable this for Apple Silicon Macs! 🤞: 

https://www.macrumors.com/2022/05/18/pytorch-gpu-accelerated-training-apple-silicon/
